ios/sdk/src/callkit/JMCallKitProxy.swift

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright @ 2018-present Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CallKit
+import Foundation
+
+/// JitsiMeet CallKit proxy
+@available(iOS 10.0, *)
+@objc public final class JMCallKitProxy: NSObject {
+    
+    override private init() {}
+    
+    // MARK: - CallKit proxy
+    
+    internal static let cxProvider: CXProvider = {
+        let config = CXProviderConfiguration(localizedName: "")
+        let provider = CXProvider(configuration: config)
+        return provider
+    }()
+    
+    internal static let cxCallController: CXCallController = {
+        return CXCallController()
+    }()
+    
+    internal static let callKitNotifier: JMCallKitNotifier = {
+        return JMCallKitNotifier()
+    }()
+    
+    internal static var cxProviderConfiguration: CXProviderConfiguration? {
```

Comment:
In `cxProvider`, `cxCallController`, and `cxProviderConfiguration` I find the `cx` prefix unnecessary. We're in the scope of "callkit" so everyone should know that when we say "provider", for example, we're talking about `CXProvider`.